game_settings.lua

```diff
@@ -1,4 +1,8 @@
 return {
     game_mode = 2,
-    last_winner_name = "[[WINNER_NAME]]"
+    last_winner_name = "[[WINNER_NAME]]",
+    mode_name = "Medium", -- Doesn't affect gameplay
+    age_score_factor = 0.8, -- Other values before this commit: 5.0, 0.25
+    research_evo_score_factor = 65, -- Other values before this commit: 100, 65
+    tech_price_multiplier = 0.35 -- Other values before this commit: 0.2, 0.75 (affects game.difficulty_settings.technology_price_multiplier during init)
```

Comment:
But we don't want to tweak this for every round, it will be confusing for players... I think they should become stable

Reply:
Also, not really understand this in the first place.